client/package.json

```diff
@@ -14,7 +14,7 @@
     "web-vitals": "^1.1.2"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "SKIP_PREFLIGHT_CHECK=true react-scripts start",
     "build": "react-scripts build",
```

Comment:
I did some digging this morning and understand this a bit more, I just sent you a chat on this too, but basically we should not set this SKIP_PREFLIGHT_CHECK flag.  To resolve the errors you were running into after installing jest and babel-jest, you need to install an older version.  This is due to a dependency on version ^26.0.0 from react-scripts.  

I didn't look too far into any added warnings on deprecations or if/when react-scripts intends to move to newer versions of jest and babel-jest, but this will at least unblock you for now.